Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
Binary files wxorig/src/generic/.DS_Store and wxnew/src/generic/.DS_Store differ
diff -uNr wxorig/src/generic/richmsgdlgg.cpp wxnew/src/generic/richmsgdlgg.cpp
--- wxorig/src/generic/richmsgdlgg.cpp	2010-08-19 20:30:14.000000000 -0400
+++ wxnew/src/generic/richmsgdlgg.cpp	2011-01-10 12:48:24.000000000 -0500
@@ -84,7 +84,7 @@
 {
     // This function can be called before the dialog is shown and hence before
     // the check box is created.
-    return m_checkBox? m_checkBoxValue : m_checkBox->IsChecked();
+    return m_checkBox? m_checkBox->IsChecked() : m_checkBoxValue;
 }
 
 #endif // wxUSE_RICHMSGDLG
